server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyConfig.scala

```diff
@@ -24,13 +25,30 @@ import sttp.tapir.server.netty.NettyConfig.EventLoopConfig
   * @param initPipeline
   *   The function to create the Netty pipeline, using the configuration instance, the pipeline created so far, and the handler which
   *   contains tapir's server processing logic.
+  *
+  * @param requestTimeout
+  *   Raises a ReadTimeoutException when no data is read from Netty within the specified period of time (in seconds) for a request.
+  *
+  * @param connectionTimeout
+  *   Specifies the maximum duration (in seconds) within which a connection between a client and a server must be established.
```

Comment:
**nitpick:** Since you switched to `FiniteDuration`, it's no longer necessarily "in seconds", so let's remove this part of the comment.